playbooks/setup_jenkins_slave.yml

```diff
@@ -42,6 +42,57 @@
         regexp: 'jenkins'
         line: 'jenkins ALL=(ALL) NOPASSWD: ALL'
 
+    - name: Copy constraints file over to cloud server
+      copy:
+        src: "{{ lookup('env', 'WORKSPACE')+'/rpc-gating/constraints.txt' }}"
```

Comment:
When concatenating strings in jinja, use ~ instead of + to ensure that the two things being concatenated are cast as strings.